src/els_config.erl

```diff
@@ -97,6 +97,8 @@ do_initialize(RootUri, Capabilities, {ConfigPath, Config}) ->
   ExcludePaths = els_utils:resolve_paths(ExcludePathsSpecs, RootPath, true),
   lager:info("Excluded OTP Applications: ~p", [OtpAppsExclude]),
   CodeReload = maps:get("code_reload", Config, disabled),
+  Runtime = maps:get("runtime", Config, #{}),
+  CtRunTest = maps:get("ct-run-test", Config, #{}),
```

Comment:
I know, but this would imply renaming also the lens itself (and the `show-behaviour-usages` one. I didn't want to do it in the same PR. Created #672 for renaming the lenses.